virttest/guest_agent.py

```diff
@@ -154,7 +154,7 @@ def __init__(self, vm, name, serial_type, serial_filename,
         :param name: Guest agent identifier.
         :param serial_type: Specific which serial type (firtio or isa) guest
                 agent will use.
-        :param serial_filename: Guest agent socket filename.
+        :param gagent_params: Guest agent params.
```

Comment:
could you make this param more clear?

Reply:
@xiagao  It means all the params the object `agent_name`, it is clear enough for me.
What info are you expect ?

Reply:
I guess xigao need a example: 
guest agent test params object, content like {'monitor_filename': file_path}

Reply:
@xiagao @xutian updated with more detailed info, please help review again, thanks!